homeassistant/components/obihai/strings.json

```diff
@@ -0,0 +1,25 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]",
+          "password": "[%key:common::config_flow::data::password%]",
+          "username": "[%key:common::config_flow::data::username%]"
+        }
+      }
+    },
+    "error": {
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]"
+    },
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
+    }
+  },
+  "issues": {
+    "manual_migration": {
+      "title": "Manual migration required for Obihai",
```

Comment:
What manual migration is required?

Reply:
Can you be more specific about what's confusing here?  This is C+P from other components that require migration from YAML.  This is just a title.  The description is... more descriptive of the process.  Would you like the title to describe how to migrate?

Reply:
The title is incorrect. There is no manual migration required (it has already migrated). It should state the YAML configuration is deprecated and should be removed.